docs/process.md

```diff
@@ -41,105 +42,65 @@ Costs are
+
+During this "Pre-RFC" phase, your proposal isn't in the RFC tracker.  It's not an RFC yet!
+
+During this phase, the proposer is responsible for moving things forward.  If
+they contact the PSC for approval to file a draft, and the PSC does not
+respond, it's the proposer who should be keeping track of that.
+
+### Draft Proposal
+
+The PSC has agreed that you should write a formal draft proposal.  You get the template document and fill it out.  You take its advice, thinking hard about what goes in each section.  Then you post it to p5p as an email with the subject "PROPOSAL:  my great idea".  Members of the list will reply with more questions and suggested amendments.  You should read them and amend the proposal to clarify your ideas or react to valid criticism.
+
+During this phase, you (the proposer) are responsible for moving things
+forward.
+
+When you think your idea has been sufficiently scrutinized, and you gotten all the feedback you're going to benefit from, submit your updated document to the PSC.  They will either request more discussion, reject the proposal, or enter it into the tracker with the status **Exploratory**.
+
+### Exploratory Status
+
+When a formal draft has been discussed sufficiently, submitted to the PSC, and is under consideration, it is entered into the proposal tracker, with the status **Exploratory**.  The PSC (or other deputized folk) will begin the process of vetting the idea.  They will review discussion about the idea, they will produce a list of questions not yet answered, and they will press existing core team members (or other experts) for input.
+
+The PSC (or deputies) will eventually either:
+ * move the document to **Implementing** status (see below) because they believe it is ready for implementation
+ * move the document to **Rejected** status because they believe it should not be implemented; This may come with advice on how to formulate an alternate proposal that has more chance of being accepted.  This isn't used for "it needs some edits", but for "it's fundamentally deeply flawed."
+ * move the document to the **Expired** status because the original proposer (or other vital personnel) are not responsive
+
+During this phase, the PSC is responsible for moving the proposal forward.
+
+### Implementing
+
+When a proposal is accepted, the PSC is stating that they'd like to see the idea implemented, and that they are likely to merge an implementation if it doesn't bring with it any unwelcome surprises or complications.
```

Comment:
Who gets to implement it? Can any core developer champion it, or should it be someone assigned by the PSC? Does a main point of contact implementation-wise needs to exist before it moves to "implementing". Can they hand it to others in case they get stuck and/or lose interest? During the implementation phase, should any issue or question arise that hasn't been foreseen in the earlier stages, who calls the shots? The proposer, the implementer, both, neither of them? does it go back to an exploratory stage?

Reply:
Also, as stated earlier in this document, _"most work on the parser and tokeniser came from just 5 people. All of them are busy, and none of them are employed by 'Perl 5 Porters'."_ Maybe getting proposers (and other interested parties) outside that circle a chance to help out could lift some of that weight, maybe even help motivate said implementors.

I understand mentoring can backfire and add another weight to the process, but it shouldn't block other people from helping attack low hanging fruits such as writing tests, documentation, or just getting to know more about how the internals work - if only to truly grasp and appreciate the incredible work the implementors are doing, and maybe in the future become implementors themselves.

Reply:
I didn't include any text because I don't think it's clear what we'd include.  The PSC *cannot assign anybody*.  That's not within our capabilities.  Suggesting it is possible would be doing everybody a disservice.  We can't just rack up great ideas, and they need to get built.  The RFC process is a way of saying "We agree this is sufficiently described to be implemented and get merged" to make implementing more attractive, but I don't think anybody has power to make implementation happen other than by *doing it*.

Reply:
> The RFC process is a way of saying "We agree this is sufficiently described to be implemented and get merged" to make implementing more attractive, but I don't think anybody has power to make implementation happen other than by _doing it_.

I like it. If this was written in the document as is, it would be enough (for me). My understanding was that the PSC would manage the implementation stage more closely, which I why I brought it up. If, once the RFC is accepted, anyone can try and implement it, then this mention (together with some minor pointers to where interested parties should direct their implementation conversation - like irc or p5p) would be nice.